doc/rst/source/supplements/module_supplements_purpose.rst_

```diff
@@ -30,6 +30,12 @@
 
 .. |mgd77track_purpose| replace:: Plot track-lines of MGD77 cruises
 
+.. |fzanalyzer_purpose| replace:: Analysis of fracture zones using crossing profiles
+
+.. |fzblender_purpose| replace:: Produce a smooth blended FZ trace
+
+.. |mlconverter_purpose| replace:: Convert chrons to ages using selected magnetic timescale
+
```

Comment:
These seem unrelated?

Reply:
yes, a personal supplement - will need to check why but removed for now.  Thanks for the fixes!